tests/test_admin/test_view.py

```diff
@@ -1418,6 +1443,90 @@ def test_post_invalid_email_error_skips_all(self):
         line_error_message = ValidationMessages.INVALID_EMAIL.format(argument=invalid_email)
         self._assert_line_message(bulk_upload_errors[0], 3, line_error_message)
 
+    @mock.patch("enterprise.utils.create_manual_enrollment_audit")
+    @mock.patch("enterprise.admin.views.create_manual_enrollment_audit")
+    @mock.patch("enterprise.models.CourseCatalogApiClient")
+    @mock.patch("enterprise.api_client.lms.EnrollmentApiClient")
+    @mock.patch("enterprise.models.EnterpriseCatalogApiClient")
+    @mock.patch("enterprise.admin.views.enroll_users_in_course")
+    @ddt.data(
+        (False, False, 200),
```

Comment:
Nit: consider writing as
```python
{'valid_course_id': False, 'course_in_catalog': False, 'expected_status_code': 200},
...
```
for additional clarity (because `ddt.unpack` will unpack a dict for you, mapping keys to parameters).

Reply:
Following up from Slack chat, for some reason using `ddt.unpack` to unpack these dicts doesn't seem to work as expected for an unknown reason:

```
TypeError: test_post_create_course_enrollments() got multiple values for argument 'valid_course_id'
```

Opting to not spend the time figuring out why in this PR.